Library/Formula/opentsdb.rb

```diff
@@ -0,0 +1,73 @@
+class Opentsdb < Formula
+  homepage "http://opentsdb.net"
+  url "https://github.com/OpenTSDB/opentsdb/releases/download/v2.1.0/opentsdb-2.1.0.tar.gz"
+  sha256 "b76e36a2d744170dde0ecd7ff3b937cbe68dd9fab9ce885aab5847de3e975157"
+
+  def install
+    ENV.deparallelize
```

Comment:
Can you please submit this issue to the upstream developers of this project and add a link to the upstream issue submission in a comment in the formula file. Thanks!